docs/events/timeline-ui-overview.asciidoc

```diff
@@ -162,3 +162,18 @@ the Timeline `ndjson` file.
 +
 NOTE: Each Timeline object in the file must be represented in a single line.
 Multiple Timeline objects are delimited with newlines.
+
+[discrete]
+[[filter-with-eql]]
+== Filter Timeline Results with EQL
+Use the *Correlations* tab to investigate Timeline results with EQL queries. You can enter EQL queries into the free-form text box under this tab and a link to EQL documentation is available in the bottom right corner of the text box.
```

Comment:
Added a link to the the EQL docs, which is also in the lower-right corner as you originally stated. But I don't think we need to point it out in the docs. 

```suggestion
Use the *Correlations* tab to investigate Timeline results with {ref}/eql.html[EQL queries]. 
```